db/seeds.rb

```diff
@@ -1,8 +1,30 @@
-# frozen_string_literal: true
+# frozen_string_literal= true
 # This file should contain all the record creation needed to seed the database with its default values.
-# The data can then be loaded with the bin/rails db:seed command (or created alongside the database with db:setup).
+# The data can then be loaded with the bin/rails db=seed command (or created alongside the database with db:setup).
 #
 # Examples:
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+   
+    teachers = Teacher.create!( 
+                  [ {     
+        first_name: Faker::Name.first_name,
+        last_name: Faker::Name.last_name,
+        email: 'random@example.com', 
+        password: 'dupablada'},
+                       {      
+        first_name: Faker::Name.first_name,
+        last_name: Faker::Name.last_name,
+        email: 'random1@example.com', 
+        password: 'dupablada1'},
+                            {
+        first_name: Faker::Name.first_name,
+        last_name: Faker::Name.last_name,
+        email: 'random3@example.com', 
+        password: 'dupablada3'}]
+                            ).find_or_create_by!(email: teacher[:email])
```

Comment:
Tried running rake db:seed and.. houston, we have a problem :D

<img width="719" alt="image" src="https://user-images.githubusercontent.com/5008759/215868781-c662bebb-51cc-4658-b774-f0a30e5b8e38.png">

Also, even if you rename `teacher[:email]` to anything other, I'm pretty sure there's no method `find_or_create_by!` on Array :) 

You will have to either:
- repeat yourself with `find_or_create_by!` or..
- figure out how to write it in a loop :)

Reply:
This was accidental. Honestly, I have no idea how that find_or_create_by method got there.  Fixed in last commit